### DISCUSSION TITLE
en/5.3.0/examples/workflows/env_from/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj-labs/hera#1072] [Website Discussions]
# en/5.3.0/examples/workflows/env_from/

https://hera.readthedocs.io/en/5.3.0/examples/workflows/env_from/

**Comment by jiantang99** [comment]
How do I use this env_from without using Container? 
I am using the method of calling steps with a python function for each step, similar to https://github.com/argoproj-labs/hera/blob/2a13cf4bf330c129509589a92787da5249fe8b10/examples/workflows/steps/callable_steps.py.
How can I use env_from in this use case?

**Reply by flaviuvadan** [reply]
You can pass it to the `@script` because `@script` and `Script` share parameters (`@script` primarily sets the `source` field, while all the other args/kwargs are shared with Script and Container)
